ludwig/datasets/text/versions.yaml

```diff
@@ -0,0 +1,3 @@
+text:
+  ohsumed: 1.0
```

Comment:
We may want to store a checksum instead.  Then we we download, we can emit a warning if the dataset has changed.

Reply:
Lets get the basics working , then we can use the checksum instead of existence like the latest code